### PR TITLE
chore: improve pull request template

### PR DESCRIPTION
**Pull request by jasonliu747**
Signed-off-by: Jason Liu <jasonliu747@gmail.com>

### Ⅰ. Describe what this PR does
Improve the PR template

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+### Ⅲ. Describe how to verify it
+
+### Ⅳ. Special notes for reviews
+
+### V. Checklist
+
+- [ ] I have written necessary docs and comments
+- [ ] I have added necessary unit tests and integration tests
+- [ ] All checks passed in `make test`
```
Comment (saintube) [inline .github/pull_request_template.md:22]: Could you please add a trailing line?

Comment (jasonliu747) [reply .github/pull_request_template.md]: sure, why not.

**Pull request merged**